action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
+runs:
+  using: "composite"
+  steps:
+    - name: Setup docker (missing on MacOS)
+      shell: bash
+      if: runner.os == 'macos'
+      run: |
+        brew install docker
+        brew install colima
+        colima start
+    - run: |
+        docker run -d --name quickstart -p 8000:8000 -e ENABLE_LOGS=true -e NETWORK=local -e ENABLE_SOROBAN_RPC=true --health-cmd "curl --no-progress-meter --fail-with-body -X POST \"http://localhost:8000/soroban/rpc\" -H 'Content-Type: application/json' -d '{\"jsonrpc\":\"2.0\",\"id\":8675309,\"method\":\"getNetwork\"}' && curl --no-progress-meter \"http://localhost:8000/friendbot\" | grep '\"invalid_field\": \"addr\"'" --health-interval 10s --health-timeout 5s --health-retries 50 stellar/quickstart:${{ inputs.version }}
```

Comment:
What's the performance like using colima on macOS 13?

Reply:
It's not great, takes about 8 minutes to start everything up, see [this run](https://github.com/stellar/stellar-cli/actions/runs/13957996459/job/39073607686)

Reply:
Oof. That's almost not worth supporting imo, especially since macos-13 is a previous generation runner and won't be available in the future.

Reply:
Yeah, but it's easy to support and it's the only Mac runner we can support anyway...

Reply:
macos-15 still uses m1 chip, we need m3 for virtualization
https://developer.apple.com/documentation/virtualization/vzgenericplatformconfiguration/isnestedvirtualizationsupported
> Nested virtualization is available for Mac with the M3 chip, and later.

The latest chip GH is _planning_ to support is M2 pro https://github.com/github/roadmap/issues/985